index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Stuart Github flow Starter</title>
+  <link rel="stylesheet" href="styles.css">
+</head>
+
+<body class="body">
+  <div class="main">
+    <h2 class="title">The Stuart Github Flow Starter</h2>
+  </div>
```

Comment:
- [ ] Great job here 💯, However. According to the requirements, you should use `<h1>` tag instead of `<h2>` because this is the main heading 👍

![image](https://user-images.githubusercontent.com/45231276/185058764-15ef94fd-5463-49cd-8df2-4b300d287d22.png)